packages/astro/src/stores/internal.ts

```diff
@@ -16,10 +16,10 @@ export const $csrState = map<{
   organization: OrganizationResource | undefined | null;
 }>({
   isLoaded: false,
-  client: null,
-  user: null,
-  session: null,
-  organization: null,
+  client: undefined,
+  user: undefined,
+  session: undefined,
+  organization: undefined,
```

Comment:
We should set the initial values to `undefined`, as it specifies that clerk-js is still being loaded